### PR TITLE
Rename class -> classes

### PR DESCRIPTION
**Pull request by danielgtaylor**
Implements refractproject/rfcs#8. Also includes some cleanup of the README and the introduction of a CHANGELOG which describes which RFCs are being implemented between versions. This also bumps the minor version of the specification.

### CODE REVIEW

**Comment by kylef** [comment]
I think we should break the changelog down and separate changes to refract itself from the included namespaces.

``` markdown
# Changelog

## 0.3.0

### Breaking

#### Refract
- The `class` meta property has been renamed to `classes`.

#### Namespaces
...
```

Just so it's clearer to the reader.

File: namespaces/api-description-namespace.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ transitions.
             {
               "element": "member",
               "meta": {
-                  "class": ["user"]
+                  "classes": ["user"]
```
Comment (pksunkara) [inline namespaces/api-description-namespace.md:253]: Sometimes we don't use indentation for value of `classes` and sometimes we do (check the `api` above). I don't have a strong opinion on this. Do we want to be consistent on this or can it be left like this?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# 0.3.0 - 2015-08-07
+
+- **BREAKING:** Renamed `class` metadata property to `classes`. [[RFC 0003](https://github.com/refractproject/rfcs/blob/master/text/0003-class-rename.md)]
```
Comment (pksunkara) [inline CHANGELOG.md]: Also, `parameters` got renamed to `hrefVariables`. And `attributes` got renamed to `data`.

Comment (pksunkara) [inline CHANGELOG.md]: And `contentTypes` got added to Transition element.

**Comment by pksunkara** [comment]
Changelog doesn't contain the full information. Otherwise lgtm.

File: namespaces/api-description-namespace.md (hunk 1 above)
Comment (danielgtaylor) [inline namespaces/api-description-namespace.md:253]: I'll create a separate PR for this :+1:

**Comment by danielgtaylor** [comment]
@kylef @pksunkara please take another look as I've addressed the feedback.

**Comment by pksunkara** [comment]
Merged. Please make a release and add the changelog there too.

**Pull request merged**